### ISSUE TITLE
Registration form doesn't load if page loaded via IPFS

### ISSUE THREAD

**Issue by andrew** [filecoin-project/slingshot#9]
When loading the slingshot registration form on https://slingshot.filecoin.io/register-now whilst you have the [IPFS browser extension](https://github.com/ipfs-shipyard/ipfs-companion) installed the page loads but the embedded form doesn't load in Firefox.

![Screenshot 2020-09-21 at 17 45 10](https://user-images.githubusercontent.com/1060/93796081-37072180-fc32-11ea-9a82-0765c66c3dba.png)

I suspect it's because the IPFS browser extension redirects to http://bafybeicwtiojksote6np6zs62fabvxrjhfm6tpukbsg23grvmuneuz5fxa.ipfs.localhost:8080/register-now/

I'd imagine that a good number of people looking to participate in slingshot have that extension installed so it'd be good to fix that or have a failure state that links out to the form on its own.

**Comment by orvn** [comment]
@andrew thanks for reporting. We're working on a fix!

**Comment by timelytree** [comment]
Hey @andrew I'm having some trouble replicating this issue on my end - can you send me a screenshot of your console? Just want to see what kinds of error might be popping up :)

**Comment by timelytree** [comment]
Hey @andrew,

So in @orvn and I's investigation, we've discovered a few things and are hoping you might have some additional insight. Here's a report of our findings:

**Firefox **
- IPFS extension _enabled_ [MacOS and Windows] in _regular_ browsing mode: website and form load **well**
- IPFS extension _enabled_ [MacOS and Windows] in _private_ browsing mode: the entire website **does not load at all**
- IPFS extension _disabled_ [MacOS and Windows] in both regular and private browsing modes, the website and form load **well**

**Chrome**
- IPFS extension _enabled_ [MacOS and Windows] in _regular_ browsing mode: website and form load **well**
- IPFS extension _disabled_ [MacOS and Windows] in both regular and private browsing modes, the website and form load **well**
- In MacOS and Windows, Chrome does not give the option to run this extension in private browsing mode

We also tested another recent website (https://network.filecoin.io/) under the same conditions as above and found the same issue in private browsing to be true. This leads me to believe that there might be some conflict with Fleek, although I can't say for sure just yet.

Ultimately, we've been unable to replicate the website loading well but the form itself not loading at all *tear I'm hoping you've got a good error message :)

As well I wanted to mention since this miiiiight be relevant, we've discovered another bug wherein the 404 page does not load correctly in production (loads just fine in development). On a route such as this: https://slingshot.filecoin.io/not-a-real-page. Expected behaviour is for the website to redirect to our internal 404 page, which does work in development. This same issue can be seen with the network.filecoin.io website. In both cases the websites are statically generated using NuxtJS and hosted on Fleek.
